cmd/state-svc/internal/resolver/resolver.go

```diff
@@ -44,6 +44,19 @@ func (r *Resolver) Version(ctx context.Context) (*graph.Version, error) {
 	}, nil
 }
 
+func (r *Resolver) AvailableUpdate(ctx context.Context) (*graph.AvailableUpdate, error) {
```

Comment:
Why not have this return the actual update information? You don't have to use it besides seeing that it's provided, but then it's there for future use.

Reply:
The additional information has been added.